stellar-contract-env-common/src/env.rs

```diff
@@ -64,11 +64,15 @@ macro_rules! call_macro_with_all_host_functions {
 
             mod context "x" {
                 {"_", fn log_value(v:RawVal) -> RawVal }
+
                 /// Get the binary contractID of the contract which invoked the
                 /// running contract. Traps if the running contract was not
                 /// invoked by a contract.
                 {"0", fn get_invoking_contract() -> Object }
                 {"1", fn obj_cmp(a:RawVal, b:RawVal) -> i64 }
+
+                {"2", fn contract_event(v:RawVal) -> RawVal }
+                {"3", fn system_event(v:RawVal) -> RawVal }
```

Comment:
My understanding from the CAP was that system_event should not be callable, and so it shouldn't be a host fn. It was described as an "internal" fn only.
```
/// Internal host function. Same as $system_event but will use ContractEventType::SYSTEM instead
func $system_event(param $val i64) (result i64)
```
Ref: https://github.com/stellar/stellar-protocol/blob/15d3738523bd577abd711aca2d1582f625d815fc/core/cap-0056.md#host-functions